### PR TITLE
Create a simple homepage

### PR DESCRIPTION
**Pull request by twoltjer**

### CODE REVIEW

**Comment by kodingkoning** [comment]
@twtduck can you take a look at the edits I made?

**Comment by twoltjer** [comment]
I like it. We should probably get an okay on this from @duckonomy before merging it in, because he may have had a vision for how the site should look (style-wise). Don't want to step on any toes!

**Comment by kodingkoning** [comment]
We also welcome content suggestions from @dschuurman

**Comment by dschuurman** [comment]
Thanks! I just put some links on our Teams page with links to articles and media resources I have made over the years - feel free to harvest these links for an "additional resources" page.
The main page should probably be simple, with a statement of our purpose and a friendly menu for resources, media, and your podcast.
Do the files have to be in the master branch to be visible? I'm assuming we will appear at https://faith-and-technology-initiative.github.io/

**Comment by twoltjer** [comment]
Yes, the site linked above is the master branch, which currently is quite empty. You can also clone and checkout (or download) a branch and just open the downloaded file in a browser. 

Having done some research, we may want to switch to the [github pages](https://guides.github.com/features/pages/) configuration for the website, which allows us to simply use markdown sources, instead of having to write all the HTML and css.

**Comment by dschuurman** [comment]
This looks good - with a few "out-of-the-box" themes to choose from... However, I will defer to others about what would work best!

File: index.html
Comment (dschuurman) [inline index.html]: Not sure CS is necessarily more abstract - some science and arts can be very abstract! Perhaps another way of saying this is: "Though computer science seems more of an artificial discipline compared to the arts or science..." 
or perhaps "Though computer science seems more preoccupied with the world of technical details than the arts or sciences..."

File: index.html
Comment (dschuurman) [inline index.html]: bring **to**

**Review by dschuurman — CHANGES_REQUESTED** [review]

File: index.html
Comment (twoltjer) [reply index.html]: Fixed, along with a couple other typos in the .md. (We can easily switch back to HTML, but this is at least easier to preview, content-wise.)

File: index.html
Comment (twoltjer) [reply index.html]: I'll go ahead and change it to the latter. Good feedback.

**Pull request merged**
